src/Stack/Package.hs

```diff
@@ -1338,6 +1334,11 @@ hpack pkgDir = do
                         , flow "was generated with a newer version of hpack,"
                         , flow "please upgrade and try again."
                         ]
+                    Hpack.ExistingCabalFileWasModifiedManually -> prettyWarnL
+                        [ cabalFile
+                        , flow "was manually modified but it is autogenerated,"
+                        , flow "please delete and modify package.yaml in the future"
```

Comment:
I think "please delete **it**" would be clearer, right now it could read as "delete + modify package.yaml".  Perhaps it's not so ambiguous, but still best to be clear as possible

Reply:
Or something along the lines of
```perl
"WARNING: $cabalFile was modified manually.  Ignoring package.yaml in favor of $cabaFile.  If you want to use package.yaml instead of $cabalFile then please delete $cabalFile."
```
?

Reply:
Hmm, perhaps!  Or it could just pre-emptively delete the cabal file! :imp:  Hah probably people wouldn't appreciate that.

Perhaps it should just error in this circumstance.  Hmm hmm, nothing seems to be quite perfect.

Reply:
One more note: The reason why I added `--force` to `hpack` and why `--force` is preferable to deleting the cabal file is, that `hpack` does some effort to minimize diffs if a cabal file is already present.  So `hpack --force` and `rm *.cabal && hpack` may lead to differently formatted cabal files.

Reply:
I'll go with @sol's verbiage, I think the hpack author is likely the best to figure these things out anyway :)